bazel/WORKSPACE

```diff
@@ -4,6 +4,11 @@ load(":deps.bzl", "deps")
 
 deps()
 
+load("@build_bazel_rules_nodejs//:index.bzl", "node_repositories")
+node_repositories(
+    node_version = "16.6.2",
```

Comment:
Bumped node to 16 since it's the [first version that supports Apple silicon](https://github.com/nodejs/TSC/issues/886).

Reply:
I moved this to `emscripten_deps.bzl`.